### DISCUSSION TITLE
Use in Wordpress without cdn

### DISCUSSION THREAD

**Discussion by myon1664** [add2cal/add-to-calendar-button#246] [Q&A] [answered]
Hi,
This is a great development! I'm using it for some time now and I am very happy with it.
Nevertheless, I would like to improve the performance of my Wordpress website by including the code on my own server (not using the provided cdn).
The Installation/Setup "Option 2: npm" doesn't look appropriate for Wordpress.
Any idea how I could achieve it?
Thank you for your attention.
Kind regards,
Christian

**Comment by jekuer** [comment]
There will be an official plugin soon! :)
If you are ok with waiting 3-5 more weeks, it will be easy as clicking the button.

**Reply by mcedav** [reply]
Hi jekuer - Any news on this plugin for wordpress? I would love to use it :-)

**Reply by jekuer** [reply]
We skipped the planned first version in favor of directly building on the upcoming V2.
It is done now, but awaiting review by the WordPress team.
In case they reject it, it will become available as direct download.
Either way, should be 5-10 business days from now on.

**Comment by jekuer** [comment, marked as answer]
Hi @myon1664 , hi @mcedav ,
good news!!
WordPress plugin is now available at https://wordpress.org/plugins/add-to-calendar-button/ .
Would love to hear your feedback and would be thankful for a good review! ❤️

**Reply by jekuer** [reply]
At the moment, being really fresh in the directory, I would recommend to search for "add2cal" if you are searching for the plugin within your WP admin panel.
